### PR TITLE
Add jayunit100 to the kubernetes-sigs org

### PR DESCRIPTION
**Pull request by jayunit100**
add jayunit100 to the kubernetes org

### CODE REVIEW

**Comment by jayunit100** [comment]
cc @justaugustus  thanks !

**Comment by mrbobbytables** [comment]
Just a quick thing - The commit msg should be something like: 
`Add jayunit100 to the kubernetes-sigs org`

For the k/org repo we really like to have a clean commit history :x

**Comment by jayunit100** [comment]
- fixed the commit !
- i think the username is in the right place?
thanks @justaugustus !

File: config/kubernetes-sigs/org.yaml [hunk 1]
```diff
@@ -176,6 +176,7 @@ members:
 - interma
 - irfanurrehman
 - janetkuo
+- jayunit100
 - javier-b-perez
```
Comment (mrbobbytables) [inline config/kubernetes-sigs/org.yaml]: The entry should be after `javier-b-perez` 👍

Comment (jayunit100) [reply config/kubernetes-sigs/org.yaml]: Haha ok will fix

**Comment by mrbobbytables** [comment]
superseded by #1704 with @jayunit100's approve 👍 
/close

**Pull request closed without merging**